Source/WebCore/contentextensions/ContentExtensionCompiler.cpp

```diff
@@ -182,8 +182,7 @@ static void addUniversalActionsToDFA(DFA& dfa, UniversalActionSet&& universalAct
     ASSERT(!root.actionsLength());
     unsigned actionsStart = dfa.actions.size();
     dfa.actions.reserveCapacity(dfa.actions.size() + universalActions.size());
-    for (uint64_t action : universalActions)
-        dfa.actions.uncheckedAppend(action);
+    dfa.actions.appendRange(universalActions.begin(), universalActions.end());
```

Comment:
Do we benefit from doing `reserveCapacity` before `appendRange?`

Reply:
At the moment, yes. However, I think we should consider optimizing Vector::appendRange() and reserve capacity at least when the input iterators are random access iterators (and std::distance() can return the size in constant time). I can look into this but likely in a follow-up.

Reply:
Following up in https://github.com/WebKit/WebKit/pull/18839.